.prettierignore

```diff
@@ -1 +1,2 @@
 .yarn
+*.mjml
```

Comment:
I know we talked in Slack about this and how the one in auth-server doesn't seem to be doing its job, does this actually remove the formatting on save? 👀 We can probably remove `*.mjml` in `fxa-auth-server` if so, it's very strange that it isn't working though and that we'd have to add it to the root ignore...

Reply:
I think it has to be in both places. From what I’ve observed on my machine, when I run from root, it auto formats the templates until I add this line. If I run from packages/fxa-auth-server, then all is good, since the `*.mjml` is already in the .prettierignore file there.

Reply:
I think it has to be in both places. From what I’ve observed on my machine, when I run from root, it auto formats the templates until I add this line. If I run from packages/fxa-auth-server, then all is good, since the `*.mjml` is already in the .prettierignore file there.